modules/forced-mfa-users/class-forced-mfa-users.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Automattic\VIP\Security\MFAUsers;
+
+class Forced_MFA_Users {
+	const MFA_SKIP_USER_IDS_OPTION_KEY = 'vip_security_mfa_skip_user_ids';
+
+	/**
+	 * The capability required to force MFA.
+	 *
+	 * @var string|array The capability slug or an array of slugs.
+	 */
+	private static $capability;
+
+	public static function init() {
+		if ( ! defined( 'VIP_SECURITY_BOOST_CONFIGS' ) ) {
+			return;
+		}
+	
+		$configs            = constant( 'VIP_SECURITY_BOOST_CONFIGS' );
+		$module_configs     = $configs['module_configs'] ?? [];
+		$forced_mfa_configs = $module_configs['forced-mfa-users'] ?? [];
+
+		self::$capability = $forced_mfa_configs['capability'] ?? [];
+		add_action( 'set_current_user', [ __CLASS__, 'filter_user_capabilities' ] );
+	}
+
+	/**
+	* Require 2FA based on capabilities set in config
+	*/
+	public static function filter_user_capabilities() {
```

Comment:
Can we have something more specific to what the function does? An idea below. 
```suggestion
	public static function maybe_require_two_factor() {
```